packages/database/src/migrations/20210805212932-AddSyncServiceLogTable-modifies-schema.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = function (db) {
+  return db.runSql(`
+    CREATE TABLE sync_service_log (
+      id TEXT PRIMARY KEY,
+      service_code TEXT NOT NULL,
+      service_type service_type NOT NULL,
```

Comment:
This is maybe unnecessary since `sync_service.code` are unique. I figured we might want to filter by `service_type` but we could also easily do that with a join.

Reply:
I like it :-)

Log tables don't need to be fully normalised - we'll probably push them into an external logging service at some point anyway